### ISSUE TITLE
GLIBC_2.28 not found (required by node) - node js Flex environement 

### ISSUE THREAD

**Issue by niciliketo** [GoogleCloudPlatform/nodejs-docker#234]
# Problem you have encountered:
As of 2022-05-13 our deploys to App Engine are failing.
This appears to be due to missing GLIBC

The build fails at this step.
```
Step #1: INFO[0036] RUN yarn install --production ||   ((if [ -f yarn-error.log ]; then       cat yarn-error.log;     fi) && false) 
Step #1: INFO[0036] cmd: /bin/sh                                 
Step #1: INFO[0036] args: [-c yarn install --production ||   ((if [ -f yarn-error.log ]; then       cat yarn-error.log;     fi) && false)] 
Step #1: node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by node)
Step #1: node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by node)
Step #1: node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by node)
Step #1: error building image: error building stage: waiting for process to exit: exit status 1
```

# What you expected to happen:
I expect the build to succeed. Last successful build was 2022-05-11

# Steps to reproduce:
- Checkout a node project that has previously been deployed to GAE.
- Run
```
gcloud app deploy --project=$GOOGLE_PROJECT
```
- NOTICE: failure with error shown above.

Other information (workarounds you have tried, documentation consulted, etc):
- Reverting to the last version of code that successfully deployed
- Checking this issue tracker - https://issuetracker.google.com
- Checking this issue tracker - https://github.com/GoogleCloudPlatform/nodejs-docker/tree/master

**Comment by jasontlouro** [comment]
@niciliketo Did you find a fix? Also experiencing this issue.

**Comment by osvald0** [comment]
Change the node [version](https://nodejs.org/en/) for the LTS instead use `>=...`.

**Comment by cjsewell** [comment]
I ran in to this today while upgrading to node 18, and update on when this could be resolved?
18.12.1 is the current LTS

**Comment by slowtick** [comment]
We could not switch to nodejs 18.x in GAE flex due to this error.

Are there plans to provide official base image supporting newer nodejs LTS or using custom image for nodejs 18+ the recommended path forward?

**Comment by kennethye1** [comment]
Yes we are working on supporting the newer runtimes in the coming weeks.

**Comment by ighunter** [comment]
Any news on support for node 18?

Node 16 is end-of-life in 1 month.  https://nodejs.org/en/blog/announcements/nodejs16-eol

**Comment by REPTILEHAUS** [comment]
Its still an issue, hitting it now.

**Comment by m-spyratos** [comment]
Any updates on this or support plans?
